### PR TITLE
Elaborated Nengo and added SpyTorch

### LINKED ISSUE #14: Add reference to SpyTorch
https://github.com/fzenke/spytorch

### PR DESCRIPTION
**Pull request by Jegp**
Added a reference to SpyTorch and elaborated on Nengo.

Closes #14

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ You can execute them by invoking the `run_*.py` scripts from the base directory.
 The default choices of hyperparameters are meant as reasonable starting points.
 
 ### Example on using the library: Long short-term spiking neural networks
-The ong short-term spiking neural networks from the paper by [G. Bellec, D. Salaj, A. Subramoney, R. Legenstein, and W. Maass](https://arxiv.org/abs/1803.09574) is one interesting way to apply norse: 
+The ong short-term spiking neural networks from the paper by [G. Bellec, D. Salaj, A. Subramoney, R. Legenstein, and W. Maass (2018)](https://arxiv.org/abs/1803.09574) is one interesting way to apply norse:
```
Comment (cpehle) [inline Readme.md]: Should be long instead of ong.

**Comment by Jegp** [comment]
I added PyNN and re-formulated the not-so-diplomatic Nengo section...

File: Readme.md [hunk 2]
```diff
@@ -98,7 +99,8 @@ The below list of projects serves to illustrate the state of the art, while expl
...
+* [Nengo DL](https://www.nengo.ai/nengo-dl/introduction.html) is a neuron simulator, and Nengo-DL is a deep learning network simulator that optimised spike-based neural networks based on an approximation method suggested by [Hunsberger and Eliasmith (2016)](https://arxiv.org/abs/1611.05141). This approach maps to, but does not build on, the deep learning framework Tensorflow, which is fundamentally different from incorporating the spiking constructs into the framework itself. In turn, this requires manual translations into each individual backend, which influences portability.
+* [PyNN](http://neuralensemble.org/docs/PyNN/) is a language for building neuronal network models. Like Nengo, PyNN is manually translating their own neural graph into a diverse set of backends, which also requires specific and exotic boilerplate code for each platform.
```
Comment (cpehle) [inline Readme.md]: I would go with something like: "PyNN is a python interface that allows you to define and simulate spiking neural network models on different backends (both software simulators and neuromorphic hardware). It does not currently provide mechanisms for optimisation or arbitrary synaptic plasticity.

Comment (Jegp) [reply Readme.md]: That sounds much better, thanks!

**Pull request merged**